ocamldoc/Makefile

```diff
@@ -227,18 +228,18 @@ DEPEND_PREREQS = $(LEXERS:.mll=.ml) \
 .PHONY: install
 install:
 	$(MKDIR) "$(INSTALL_BINDIR)"
-	$(MKDIR) "$(INSTALL_LIBDIR)/ocamldoc"
+	$(MKDIR) "$(INSTALL_OCDOCDIR)"
 	$(INSTALL_PROG) $(OCAMLDOC) "$(INSTALL_BINDIR)"
 	$(INSTALL_DATA) \
 	  ocamldoc.hva *.cmi $(OCAMLDOC_LIBCMA) \
-	  "$(INSTALL_LIBDIR)/ocamldoc"
+	  "$(INSTALL_OCDOCDIR)"
```

Comment:
Can I slightly bike-shed the name? `INSTALL_LIBDIR_OCAMLDOC` or `INSTALL_OCAMLDOC_LIBDIR` - it's mildly more readable to me to be able to see from the name that we're still installing to something under `INSTALL_LIBDIR` where `OCDOCDIR` sounds more like something being installed in `INSTALL_DOCDIR`.